lib/components/result-view/result-view.js

```diff
@@ -131,12 +130,12 @@ class ResultViewComponent extends React.Component {
                 className="icon icon-file-symlink-file"
                 onClick={this.openInEditor}
               />
-              <div
-                className={`icon icon-${this.expanded.get()
-                  ? "fold"
-                  : "unfold"}`}
-                onClick={this.toggleExpand}
-              />
+              {this.el.scrollHeight > this.el.clientHeight
```

Comment:
This won't work, since `this.el` can be null (see [flow type errors](https://travis-ci.org/nteract/hydrogen/jobs/239749187#L140)). We need to guard against that case.

I can recommend the [`flow-ide`](https://github.com/steelbrain/flow-ide) package for seeing the flow type-checking result directly in Atom.